app/Http/Middleware/Authenticate.php

```diff
@@ -22,7 +22,11 @@ public function handle($request, Closure $next, $guard = null)
             if ($request->ajax() || $request->wantsJson()) {
                 return response('Unauthorized.', 401);
```

Comment:
[rank B] what does "401" means? Please refer to other about this. I did raise this issue before.